package.json

```diff
@@ -46,11 +46,11 @@
     "markdown-it-linkify-images": "^1.1.1",
     "markdown-it-mark": "^2.0.0",
     "markdown-it-regexp": "^0.4.0",
+    "markdown-it-sub": "^1.0.0",
+    "markdown-it-sup": "^1.0.0",
```

Comment:
😮

Reply:
that's strange.. i don't remember installing these. uninstalled them now 👍

Reply:
~that's strange.. i don't remember installing these. uninstalled them now~

I hadn't rebased my code on to master

Reply:
rectified